Doc/library/bisect.rst

```diff
@@ -210,7 +210,7 @@ records in a table::
     >>> Movie = namedtuple('Movie', ('name', 'released', 'director'))
 
     >>> movies = [
-    ...     Movie('Jaws', 1975, 'Speilberg'),
+    ...     Movie('Jaws', 1975, 'Spielberg'),
     ...     Movie('Titanic', 1997, 'Cameron'),
     ...     Movie('The Birds', 1963, 'Hitchcock'),
     ...     Movie('Aliens', 1986, 'Scott')
```

Comment:
Ridley Scott directed the original [_Alien_](https://en.wikipedia.org/wiki/Alien_(film)) (1979) but James Cameron directed the sequel [_Aliens_](https://en.wikipedia.org/wiki/Aliens_(film)) (1986).
```suggestion
    ...     Movie('Aliens', 1986, 'Cameron')
```